plugins/parsers/influx/influx_upstream/parser.go

```diff
@@ -104,7 +104,8 @@ func convertToParseError(input []byte, rawErr error) error {
 type Parser struct {
 	DefaultTags map[string]string `toml:"-"`
 	// If set to "series" a series machine will be initialized, defaults to regular machine
-	Type string `toml:"-"`
+	Type       string `toml:"-"`
+	ParserType string `toml:"influx_parser_type"`
```

Comment:
I think this is not needed, instead add back `"influx_parser_type"` back to [` missingTomlField()` here](https://github.com/influxdata/telegraf/blob/21a7d6fcbe8cfdee18fe62518575bd4b0446d223/config/config.go#L1236)...

Reply:
reverted